include/Dialects/LinalgExt/LinalgExtOps.td

```diff
@@ -264,20 +264,9 @@ def LinalgExt_PerformConcurrentlyOp : LinalgExt_PureOp<"perform_concurrently", [
   // TODO(apaszke, ntv): Add an interface for ops that can appear inside
   // perform_concurrently.
   let extraClassDeclaration = [{
-    SmallVector<Type> yieldedTypes() {
-      SmallVector<Type> ret;
-      for (Operation& op : *getBody()) {
-        if (auto sliceOp = llvm::dyn_cast<ParallelInsertSliceOp>(op)) {
-          ret.push_back(sliceOp.yieldedType());
-          continue;
-        }
-        if (auto endPerformOp = llvm::dyn_cast<EndPerformConcurrentlyOp>(op)) {
-          continue;
-        }
-        llvm_unreachable("Unexpected operation in perform_concurrently");
-      }
-      return ret;
-    };
+    SmallVector<Type> yieldedTypes();
+    SmallVector<Value> yieldedValues();
```

Comment:
I'm not sure if yielded values are even well defined, so I'd rather not add it unless we have to.